### PR TITLE
GHCR.io image push - 8.*

### PR DESCRIPTION
**Pull request by Praveenraj-K**
Added
- Pushing the Docker images to ghcr.io (Git Container Registry)
- Also pushed in only 8.3, 8.1 branches

### CODE REVIEW

File: .github/workflows/build-and-publish.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 name: Build and Publish
 
 on: 
-  push:
+  push: 
+    branches: ['8.3']
```
Comment (briskt) [inline .github/workflows/build-and-publish.yml]: I think we still need tags on the 8.1 branch

**Review by briskt — CHANGES_REQUESTED** [review]

File: .github/workflows/build-and-publish.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: Build and Publish
 
 on: 
   push: 
-    branches: ['8.3']
+    #branches: ['8.3']
```
Comment (briskt) [inline .github/workflows/build-and-publish.yml]: Please remove it entirely.

File: .github/workflows/build-and-publish.yml (hunk 1 above)
Comment (Praveenraj-K) [reply .github/workflows/build-and-publish.yml]: removed the branch config

File: .github/workflows/build-and-publish.yml (hunk 2 above)
Comment (briskt) [reply .github/workflows/build-and-publish.yml]: I'm sorry I changed my mind. You were right to limit it. Please change it to `[ '8.*' ]`

Comment (Praveenraj-K) [reply .github/workflows/build-and-publish.yml]: Its done now. Please check and let me know.

**Review by briskt — APPROVED** [review]

**Pull request merged**